crates/dbs-arch/src/aarch64/gic/gicv2.rs

```diff
@@ -52,10 +48,6 @@ impl GICv2 {
 }
 
 impl GICDevice for GICv2 {
-    fn version() -> u32 {
```

Comment:
Is there any special attention to adjusting the position of the version() function?

Reply:
It's to keep the same order with trait declaration.